judgments/views.py

```diff
@@ -100,9 +100,13 @@ def advanced_search(request):
         "order": params.get("order", ""),
         "from": params.get("from"),
         "to": params.get("to"),
+        "per_page": params.get("per_page"),
     }
     page = params.get("page", 1)
     page = page if page else 1
+    per_page = (
+        params.get("per_page") if params.get("per_page") else str(RESULTS_PER_PAGE)
```

Comment:
Might be worth putting a sensible maximum on the page size -- it feels like a good way of asking for literally everything, which might make denial of service a possibility. (I'm not convinced this is actually a problem, but best to be proactive.)

Reply:
aha yep good shout - i will take a look!